tests/logout.test.js

```diff
@@ -0,0 +1,33 @@
+
+import loginPage from '../pages/login.page';
+import logoutPage from '../pages/logout.page';
+
+describe('OpenBank', () => {
+
+    it('Should show logout success message when clicking on logout menu', async () => {
+
+      await loginPage.open('login');
+      let username = 'jsmith@demo.io';
+      let password = 'Demo123!';
+      await loginPage.logIn(username, password);
```

Comment:
This logic seems to be a pre-requisite for all logout tests, and it's currently duplicated. Consider using a `beforeEach()` **hook** to put it there, so you define it once, and it'll run automatically before each test in the suite, removing this duplication.

Reply:
Done